bin/checkCollectiveAccess

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+cautils=`which caUtils`
+if [ ! -f $cautils ]; then
+    echo "You need to have caUtils from the CollectiveAccess installation directory on your PATH"
+    echo "  it should be a file at /support/bin/caUtils relative to your providence installation"
+    exit 1
+fi
+if [ ! -d $COLLECTIVEACCESS_HOME ]; then
```

Comment:
to avoid unbound variable check this needs to be `[ -z "${COLLECTIVEACCESS_HOME-}" ]`, fixed now